validator/validator-main.protoascii

```diff
@@ -4429,6 +4429,24 @@ tags: {
   tag_name: "LEGEND"
 }
 
+# 4.11 (HTML5.3) Interactive elements
+# 4.11.1 (HTML5.3) The details element
+tags: {
+  html_format: AMP
```

Comment:
Can you add the other formats here as well:

- AMP4ADS
- AMP4EMAIL
- EXPERIMENTAL

Reply:
Gregable checked with each spec owner so this change is okay for all of these formats.

Reply:
Looks good for AMP4ADS